### DISCUSSION TITLE
Static library

### DISCUSSION THREAD

**Discussion by yanchoyanev** [Kistler-Group/sdbus-cpp#175] [Ideas]
The code is not that much and may make sense to be used as a static library. Can we have an included static library to the -dev package and an option whether to use static or dynamic linking?

**Comment by sangelovic** [comment]
You can build sdbus-c++ as a static library. As for the option -- simply set `-DBUILD_SHARED_LIBS=OFF` configuration option when configuring the library (https://cmake.org/cmake/help/latest/variable/BUILD_SHARED_LIBS.html). As for the dev package, you probably have to contact maintainers of your package system to discuss the contents of sdbus-c++ dev package.
